internal/pkg/describe/webapp.go

```diff
@@ -34,8 +40,35 @@ type TaskSize struct {
 	Memory string
 }
 
+// SerializedWebAppConfig contains serialized configuration parameters for a web application.
+type SerializedWebAppConfig struct {
+	Environment string `json:"environment"`
+	Port        string `json:"port"`
+	Tasks       string `json:"tasks"`
+	CPU         string `json:"cpu"`
+	Memory      string `json:"memory"`
+}
+
+// SerializedWebAppRoute contains serialized route parameters for a web application.
+type SerializedWebAppRoute struct {
+	Environment string `json:"environment"`
+	URL         string `json:"url"`
+	Path        string `json:"path"`
+}
+
+// SerializedWebApp contains serialized parameters for a web application.
+type SerializedWebApp struct {
+	AppName        string                    `json:"appName"`
+	Type           string                    `json:"type"`
+	Project        string                    `json:"project"`
+	Configurations []SerializedWebAppConfig  `json:"configurations"`
+	Routes         []SerializedWebAppRoute   `json:"routes"`
+	Resources      map[string][]*CfnResource `json:"resources,omitempty"`
+}
```

Comment:
Should we remove the word `Serialized` from these constructs?

Reply:
I personally prefer to keep `Serialized` because as you might notice that we have `WebAppECSParams` struct with very similar member variable set as `SerializedWebAppConfig`. The former struct is when we fetch those parameters while the latter is used for output, and `Serialized` identifier makes the usage for those structs stand out.

Reply:
I disagree because serialization is a form the data might take from the structure, so I find including it in the name confusing, but it's not a big deal.

Reply:
Oh I get your point :+1: `Serialized` is not a good one.